### PR TITLE
main.sh: update config output to add dashboard

### PR DESCRIPTION
**Pull request by maryamtahhan**

### CODE REVIEW

**Comment by rootfs** [comment]
@SamYuan1990 @vprashar2929 can we remove microshift test? It looks most test cases are similar with kind

File: main.sh [hunk 1]
```diff
@@ -166,21 +166,21 @@ ebpf() {
 		mkdir -p /tmp/elfutils-source
 		cd /tmp/elfutils-source
 		yumdownloader --source elfutils
-		rpm2cpio elfutils-0.189-3.el9.src.rpm | cpio -iv
+		rpm2cpio elfutils-0.190-2.el9.src.rpm | cpio -iv
```
Comment (SamYuan1990) [inline main.sh:169]: @rootfs , @maryamtahhan to keep consistent with https://github.com/sustainable-computing-io/kepler/blob/main/build/Dockerfile.builder#L11 
please considering rollback the change for libbpf or bump up libbpf version in kepler build image.
either fine for me.

File: main.sh [hunk 2]
```diff
@@ -122,7 +122,7 @@ print_config() {
 
 		Monitoring
 		  * Install Prometheus : $prom_install_msg
-		  * Install Grafana    : $GRAFANA_ENABLE
+		  * Install Grafana + Kepler dashboard   : $GRAFANA_ENABLE
```
Comment (SamYuan1990) [inline main.sh:125]: IMO, if `Install Grafana + Kepler dashboard`'s parameter as GRAFANA_ENABLE it looks strange.

**Comment by SamYuan1990** [comment]
> @SamYuan1990 @vprashar2929 can we remove microshift test? It looks most test cases are similar with kind

Previously the microshift is setup by @sthaha , either fine for me, I will leave to @sthaha

File: main.sh (hunk 1 above)
Comment (sthaha) [reply main.sh:169]: Absolutely agree with @SamYuan1990 , lets go in lock step with kepler dependencies.

**Comment by sthaha** [comment]
> Previously the microshift is setup by @sthaha 

s/sthaha/vprashar2929 :-)

File: main.sh (hunk 1 above)
Comment (maryamtahhan) [reply main.sh:169]: I update these as they were causing the unit tests to fail - so i imagine the Kepler build image needs to be  updated. I will push a PR to kepler

File: main.sh (hunk 2 above)
Comment (maryamtahhan) [reply main.sh:125]: so are we saying we want a separate flag for the Kepler dashboard? or maybe I didn't understand the original intention in https://github.com/sustainable-computing-io/local-dev-cluster/pull/79#pullrequestreview-2032883413 
to change the output hint  :(

File: main.sh (hunk 1 above)
Comment (maryamtahhan) [reply main.sh:169]: created PR https://github.com/sustainable-computing-io/kepler/pull/1408

**Pull request merged**